torch/_torch_docs.py

```diff
@@ -1465,6 +1465,69 @@ def merge_dicts(*dicts):
     tensor([ 0.5000, -0.4702, -0.4599,  0.5000])
 """.format(**common_args))
 
+add_docstr(torch.complex,
+           r"""
+complex(real, imag, *, out=None) -> Tensor
+
+Constructs a complex tensor with its real part equal to :attr:`real` and its
+imaginary part equal to :attr:`imag`.
+
+.. math::
+    \text{out} = \text{real} + \text{imag} \cdot i
+""" + r"""
+Args:
+    real (Tensor): The real part of the complex tensor. Must be
```

Comment:
"Must be a float or double tensor." might be a more natural sounding second sentence.

Reply:
After reading it a few more times I think "Must be float or double." is even better.